Pyjion/pycomp.h

```diff
@@ -173,6 +166,11 @@
 
 #define METHOD_INT_TO_FLOAT					    0x00000072
 
+#define METHOD_STOREMAP_NO_DECREF_TOKEN			0x00000073
+#define METHOD_FORMAT_VALUE						0x00000074
```

Comment:
The formatting in this file seems to gotten a little funky for some reason. Tabs?